packages/nuxt/src/app/components/nuxt-island.ts

```diff
@@ -104,12 +104,18 @@ export default defineComponent({
       }
     }
 
-    const payloadSlots: NonNullable<NuxtIslandResponse['slots']> = {}
-    const payloadComponents: NonNullable<NuxtIslandResponse['components']> = {}
+    const payloads: {
+     slots: NonNullable<NuxtIslandResponse['slots']>
+     components:  NonNullable<NuxtIslandResponse['components']>
+    } = {
```

Comment:
we could probably extract this into a type alias above, or simplify to:

```suggestion
    const payloads: Required<Pick<NuxtIslandResponse, 'slots' | 'components'>> = {
```

Reply:
oops indeed 😅